lms/djangoapps/teams/static/teams/js/views/instructor_tools.js

```diff
@@ -25,14 +29,46 @@
 
                 deleteTeam: function (event) {
```

Comment:
Please add Jasmine tests for team deletion:
1. Can delete a team
2. Can cancel deletion of a team
3. Handles an error when deleting a team

There is some controversy about when to write Jasmine versus Bok Choy, and you already have Bok Choy tests here. My opinion is that we want good Jasmine code coverage, and it isn't possible to fake out server errors etc in Bok Choy. We also should follow the testing pyramid of preferring unit tests over integration tests:

https://github.com/edx/edx-platform/blob/master/docs/en_us/internal/testing.rst